var/spack/repos/builtin/packages/netcdf/package.py

```diff
@@ -131,10 +151,18 @@ def install(self, spec, prefix):
             LDFLAGS.append("-L%s/lib"     % spec['szip'].prefix)
             LIBS.append("-l%s"         % "sz")
 
+        # PnetCDF support
+        if '+parallel-netcdf' in spec:
+            config_args.append('--enable-pnetcdf')
+            config_args.append('CC=%s' % spec['mpi'].mpicc)
```

Comment:
Also, out of curiosity, what exactly is the difference between `netcdf+mpi` and `netcdf+parallel-netcdf`? I've never used either, but my users regularly use serial `netcdf` and `parallel-netcdf`.